src/main/java/org/jooq/lambda/Seq.java

```diff
@@ -283,18 +316,37 @@
         List<? extends U> list = other.toList();
 
         return flatMap(t -> seq(list)
-              .filter(u -> predicate.test(t, u))
-              .onEmpty(null)
-              .map(u -> tuple(t, u)))
-              .onClose(other::close);
+                           .filter(u -> predicate.test(t, u))
+                           .onEmpty(null)
+                           .map(u -> tuple(t, u)))
+                           .onClose(other::close);
+    }
```

Comment:
Thanks for fixing the formatting. Perhaps, this could be done in a separate task as well. In particular, now, the `onClose()` method is not aligned correctly according to your fix's intention.